backend/app/services/knowledge/structured/sql_validator.py

```
@@ -0,0 +1,209 @@
+        "TRUNCATE",
+        "GRANT",
+        "REVOKE",
+        "EXECUTE",
+        "EXEC",
+        "MERGE",
+        "REPLACE",
+        "LOAD",
+        "COPY",
+        "ATTACH",
+        "DETACH",
+    ]
```

Ruff (0.15.0)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)